### ISSUE TITLE
Add Spell-Checker to linter

### ISSUE THREAD

**Issue by kulla** [vroland/mwlint#21]

**Comment by vroland** [comment]
I don't think this is a good idea, since spell checking preferences may be very different for different people. Furthermore, shipping a complete dictionary with the linter when loading the editor would be a lot of bloat. 
I would rather rely on the user's browser for spell checking.

**Comment by kulla** [comment]
Yes, I meant to activate the browsers chell specker inside the editor :-) (which is currently not the case).
